src/main/resources/processes/DCU_MIN_MARKUP.bpmn

```diff
@@ -175,11 +175,6 @@
       <bpmn:incoming>SequenceFlow_1r3bhri</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_14u7a2d</bpmn:outgoing>
     </bpmn:serviceTask>
-    <bpmn:serviceTask id="ServiceTask_1wwfdzr" name="Update Teams for Topic" camunda:expression="${bpmnService.updateTeamsForPrimaryTopic(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey, execution.getVariable(&#34;Topics&#34;), &#34;DCU_MIN_INITIAL_DRAFT&#34;, &#34;DraftingTeamUUID&#34;, &#34;DraftingTeamName&#34; )}">
```

Comment:
I'm not sure why this has been removed.

Reply:
I'm not sure. I never intentionally deleted it, but I have replaced it.